src/components/Products/ProductsListItem.vue

```diff
@@ -1,103 +1,103 @@
 <template>
-	<div class="product-list-item">
-		<div
-				class="product-list-item__picture"
-				:style="{ backgroundImage: 'url(' + imgUrl + ')' }"
-		></div>
-		<div class="product-list-item__description">
+  <div class="product-list-item">
+    <div
+      class="product-list-item__picture"
+      :style="{ backgroundImage: 'url(' + imgUrl + ')' }"
+    ></div>
+    <div class="product-list-item__description">
       <span
-		      class="product-list-item__description__title"
-		      v-if="name.length < 20"
-      >{{ name }}
+        class="product-list-item__description__title"
+        v-if="name.length < 20"
+        >{{ name }}
       </span>
-			<span class="product-list-item__description__title" v-else>
+      <span class="product-list-item__description__title" v-else>
         {{ name.substring(0, 14).concat("..") }}
```

Comment:
this supposed to be text overflow? If yes then what in case 4 characters long name? It will also display `...`. This can be done with pure css approach.